### PR TITLE
[Koh Yi Da] Duke Increments

### PR DESCRIPTION
**Pull request by kohyida1997**

### CODE REVIEW

**Review by fabbbbbbyy — COMMENTED** [review]
Your code is clean and simple to read. One suggestion would be to abstract out long blocks of code from the public methods into private methods.

File: src/main/java/duke/storage/Storage.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    public void setChangedTrue() {
+        storageIsChanged = true;
+    }
+
+    /**
+     * Returns a true or false indicating whether storage has been updated.
+     *
+     * @return Status of storage.
+     */
+    public boolean storageUpdated() {
```
Comment (tysg) [inline src/main/java/duke/storage/Storage.java]: Will it be better to name the method to `isUpdated`?

File: src/main/java/duke/datetime/Timing.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package duke.datetime;
+
+/**
+ * Represents a timing in the 24-hour clock format.
+ */
+public class Timing {
```
Comment (tysg) [inline src/main/java/duke/datetime/Timing.java:6]: Love the implementation here!

File: src/main/java/duke/datetime/Date.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package duke.datetime;
+
+/**
+ * Represents a date in the format Day/Month/Year.
+ */
+public class Date {
```
Comment (tysg) [inline src/main/java/duke/datetime/Date.java:9]: I'm not sure if you have already considered this, but would it be better to use the built-in Java Date class?

**Review by tysg — COMMENTED** [review]
Hi Yi Da, the code LGTM. Very well composed and cleanly written!

**Pull request closed without merging**